src/modules/clustering/hierarchical_clustering_method.cpp

```diff
@@ -223,6 +223,7 @@ std::vector<Cluster> hierarchical_clustering_method(std::vector<Junction> const
         // Add new clusters: junctions with the same label belong to one cluster
         for (auto & [lab, jun] : label_to_junctions )
         {
+            (void) lab;
```

Comment:
does this solve a warning?

Reply:
`lab` is otherwise unused, and you cannot use `[[maybe_unused]]` with structural bindings. Using `std::get<1>` would also be possible, but it doesn't make a difference, and the structural binding is easier to understand.

Reply:
ah thx for the explanation :)